### ISSUE TITLE
Allow non-voice users to play music

### ISSUE THREAD

**Issue by lanerussell** [Cog-Creators/Red-DiscordBot#6195]
### Type of feature request

Command

### Description of the feature you're suggesting

It would be nice if Red could be configured to allow users who aren't connected to voice to use the `!play` (and other) commands. Currently, the user just sees the following error:
```
Unable To Play Tracks
Connect to a voice channel first.
```

We're preparing to have a small get-together with Red playing music on a laptop hooked up to the stereo. It'd be nice if folks could play music from their mobile apps without joining voice.

**Comment by cens0r3d** [comment]
i’ll be with u on this, we are djing sometimes for a twitch streamer but need to join the channel to play some music..

**Comment by Dav-Git** [comment]
Counter argument: what if I am listening to music in a channel with some people and (as sadly on discord is often the case) some random i***t messes with our music. Or even worse puts on disturbing noises that will negatively effect everyone in that VC.

Having to be in the voice channel seems like a high enough barrier to entry to stop a large portion of these users.

**Comment by Kowlin** [comment]
> Counter argument: what if I am listening to music in a channel with some people and (as sadly on discord is often the case) some random i***t messes with our music. Or even worse puts on disturbing noises that will negatively effect everyone in that VC.

At the very least to be accepted as a PR I'd like to see this as a toggle-able settings as it brings the complications you mentioned.  This is one of those cases where the needs of some users far outgrew the needs of a regular user. And while that's fine, to introduce this change for everyone isn't needed. And may be even considered undesirable.

**Comment by lanerussell** [comment]
Yep, I totally agree that such functionality could open a path for bad actors. That's why the initial feature request specifies:

> It would be nice if Red could be **configured** to allow users who aren't connected to voice to use the `!play` (and other) commands.

**Comment by cens0r3d** [comment]
> Counter argument: what if I am listening to music in a channel with some people and (as sadly on discord is often the case) some random i***t messes with our music. Or even worse puts on disturbing noises that will negatively effect everyone in that VC.
> 
> Having to be in the voice channel seems like a high enough barrier to entry to stop a large portion of these users.

for that reason ill use the dj role ;)
